Libraries/LibTLS/TLSv12.cpp

```diff
@@ -519,6 +597,28 @@ void TLSv12::ensure_hmac(size_t digest_size, bool local)
         m_hmac_remote = move(hmac);
 }
 
+bool Certificate::is_valid() const
+{
+    auto now = Core::DateTime::now();
+    auto now_string = String::format("%04d%02d%02d%02d%02d%02dZ", now.year(), now.month(), now.weekday(), now.hour(), now.minute(), now.second());
+
+    if (!not_before.is_empty()) {
+        if (strcasecmp(not_before.characters(), now_string.characters()) > 0) {
```

Comment:
This is pretty awkward. Can we add something to `Core::DateTime` so we don't have to compare dates by converting them to strings first? :)

Reply:
an alternative would be something like strptime...
or we could just wrap this string compare in core::datetime, which is...quite meh

Reply:
Hiding the ugliness inside `Core::DateTime` is better IMO because that at least makes this logic look better. Then we can go and fix up the datetime code later :)